rules/power-point-comments/rule.md

```diff
@@ -0,0 +1,46 @@
+---
+type: rule
+title: Do you use PowerPoint comments for feedback?
+uri: power-point-comments
```

Comment:
Hi @synnoveisabel 

- [ ] 1. Can you update the URI to have "powerpoint" in 1 word... without the dash 

Note: You need to change it in 3 places:
* URI
* folder name
* category index